lib/commands/performance.js

```diff
@@ -0,0 +1,297 @@
+import { errors } from 'appium-base-driver';
+import { sleep } from 'asyncbox';
+import _ from 'lodash';
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.postDismissAlert = async function () {
+  throw new errors.NotYetImplementedError();
+};
+
+const SUPPORTED_PERFORMANCE_DATA_TYPES = {
+  'cpuinfo': 'the amount of cpu by user and kernel process - cpu information for applications on real devices and simulators',
+  'memoryinfo': 'the amount of memory used by the process - memory information for applications on real devices and simulators',
+  'batteryinfo': 'the remaining battery power - battery power information for applications on real devices and simulators',
+  'networkinfo': 'the network statistics - network rx/tx information for applications on real devices and simulators'
+};
+
+commands.getPerformanceDataTypes = function () {
+  return _.keys(SUPPORTED_PERFORMANCE_DATA_TYPES);
+};
+
+commands.getPerformanceData = async function (applicationPackageName, performanceDataType, performanceDataReadTimeout) {
+
+  if (performanceDataType === "batteryinfo")
```

Comment:
ok, I tried, 
I have faced below issue when I try to commit the code.

hwanghuiseon-ui-MacBook-Pro:appium-android-driver hwangheeseon$ git commit -m "change to include braces"
Pre-commit checks...
[21:27:50] Using gulpfile ~/work/appium-android-driver/gulpfile.js
[21:27:50] Starting 'eslint'...
[21:28:05]
/Users/hwangheeseon/work/appium-android-driver/lib/commands/performance.js
  27:8  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  30:8  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  33:8  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  37:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style

✖ 4 problems (4 errors, 0 warnings)

[21:28:05] 'eslint' errored after 16 s
[21:28:05] ESLintError in plugin 'gulp-eslint'
Message:
    Failed with 4 errors
pre-commit:
pre-commit: We've failed to pass the specified git pre-commit hooks as the `lint`
pre-commit: hook returned an exit code (1). If you're feeling adventurous you can
pre-commit: skip the git pre-commit hooks by adding the following flags to your commit:
pre-commit:
pre-commit:   git commit -n (or --no-verify)
pre-commit:
pre-commit: This is ill-advised since the commit is broken.
pre-commit: